boards/matek/h743/init/rc.board_sensors

```diff
@@ -20,3 +20,9 @@ dps310 -I start -a 118
 
 # External mag
 qmc5883l -X start -a 13
+
+# External airspeed sensor/differntial pressure
+if param compare -s SENS_EN_MS4525DO 1
```

Comment:
I was saying you should be able to skip this entirely and rely on the central startup here.

https://github.com/PX4/PX4-Autopilot/blob/2e1cdc9e759031784c6f5ef8ddccd63b2cde6f26/ROMFS/px4fmu_common/init.d/rc.sensors#L145-L149

Then we can either update the MS4525DO driver to scan all possible I2C addresses.

https://github.com/PX4/PX4-Autopilot/blob/2e1cdc9e759031784c6f5ef8ddccd63b2cde6f26/src/drivers/differential_pressure/ms4525do/MS4525DO.hpp#L57

Or short term we can simply try address 0x40 here if the default start fails. https://github.com/PX4/PX4-Autopilot/blob/2e1cdc9e759031784c6f5ef8ddccd63b2cde6f26/ROMFS/px4fmu_common/init.d/rc.sensors#L148